src/components/pages/my-space.js

```diff
@@ -59,6 +62,13 @@ function MySpace({ initialState, PageTemplate }) {
           <TabPane className="Tabs-tabPane" tab={t('accountTabTitle')} key="3">
             <AccountTab formItemLayout={formItemLayout} tailFormItemLayout={tailFormItemLayout} />
           </TabPane>
+          {user.storage && (
+            <TabPane className="Tabs-tabPane" tab={t('common:storage')} key="4">
+              <h5>{`"${storagePlan.name}" ${t('storagePlanLabel')}`}</h5>
```

Comment:
I think at a point in the near future (when we can remove storage plans from users), there might be the scenario where:
* `storagePlan` is `null`
* `user.storage` is smth like `{ plan: null, usedStorageInBytes: 184637, reminders: [] }`

I'd suggest to show something like "No storage plan" here in case `storagePlan` is not defined.

Reply:
Treated `storagePlan === null`, thanks